spec.bs

```diff
@@ -6,57 +6,417 @@ Status: CG-DRAFT
 Group: WICG
 URL: https://github.com/WICG/shared-storage
 Editor: Camillia Smith Barnes, Google https://google.com, cammie@chromium.org
+Markup Shorthands: markdown yes
 Abstract: Shared Storage is a storage API that is intentionally not partitioned by top-frame site (though still partitioned by context origin of course!). To limit cross-site reidentification of users, data in Shared Storage may only be read in a restricted environment that has carefully constructed output gates.
 </pre>
+<pre class=link-defaults>
+spec:url; type:interface; text:URL
+</pre>
+<style>
+/* .XXX from https://resources.whatwg.org/standard.css */
+.XXX {
+  color: #D50606;
+  background: white;
+  border: solid #D50606;
+}
+</style>
 
 Introduction {#intro}
```

Comment:
Some general style suggestions:

- Use |camelCase| for temporary variables
- Defined types and fields should be <dfn>spaced out naturally</dfn>
- When you define something that is associated with another thing, use <dfn for="shared storage data">per origin data</dfn>, and access it via: Let |perOriginData| be |sharedStorageData|’s [=shared storage data/per origin data=].

Reply:
Ok, I think that I did this, more or less.  But please let me know if you want me to give every definition a "for" field (which seems a bit excessive to do everywhere).